src/Damage.js

```diff
@@ -51,7 +57,13 @@ class Damage {
    */
   evaluate(target) {
     let amount = this.amount;
+
     if (this.attacker) {
+      const critChance = Math.max(this.attacker.getMaxAttribute('critical') || 0, 0);
```

Comment:
I'm really really torn on having this part of core. I lean to leaving it in simply because this seems to degrade gracefully, insofar as if the character doesn't have the critical attribute it just never does a critical hit.